src/scripts/gym/Gym.ts

```diff
@@ -70,13 +70,16 @@ class Gym extends TownContent {
         {
             quest = true,
             achievement = true,
-        }: gymFlags = {}
+        }: gymFlags = {},
+        public displayName?: string
```

Comment:
Can we use the displayName for all the `...2` gym leaders? Then we can remove all the `.replace(/\d/g, '')` code we have